node/mock-test.ts

```diff
@@ -137,8 +155,14 @@ export class MockTestRunner {
         }
     }
 
+    public run(nodeVersion?: number): void {
```

Comment:
[Suggestion] We can add `@deprecated` tag here
https://jsdoc.app/tags-deprecated.html

Reply:
done